src/backends/plonky2/mock/signedpod.rs

```diff
@@ -43,15 +43,11 @@ pub struct MockSignedPod {
     kvs: HashMap<Key, Value>,
 }
 
-// impl MockSignedPod {
-//     pub fn deserialize(id: PodId, signature: String, dict: Dictionary) -> Self {
-//         Self {
-//             id,
-//             signature,
-//             dict,
-//         }
-//     }
-// }
+impl MockSignedPod {
+    pub fn deserialize(id: PodId, signature: String, kvs: HashMap<Key, Value>) -> Self {
```

Comment:
this looks like a constructor with a different name :P 
Will this method be used externally?  If not, can we make it `pub(crate)`?

Reply:
It does have to be public if we want users of the library to be able to deserialize SignedPods.

Reply:
Won't users of the library interface the library through the frontend?  And deserialize `frontend::SignedPod` (which internally deserializes `MockSignedPod`)?

Reply:
Oh, sorry, you're quite right. I'll make it `pub(crate)`.